examples/web/index.html

```diff
@@ -80,6 +80,11 @@ <h2>Button</h2>
           <button type="button" class="Button Button--primary">Primary button</button>
           <button type="button" class="Button Button--secondary">Secondary button</button>
           <button type="button" class="Button Button--tertiary">Tertiary button</button>
+          <button type="button" class="Button Button--inverted">Inverted button</button>
```

Comment:
Could we have this variant on dark background? Maybe `.example-box` would help?

<img width="306" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/156625225-8816cf47-7e5d-4cbd-935d-8f3153520891.png">